rkt/stage1hash.go

```diff
@@ -170,18 +171,20 @@ func getLocalDefaultStage1Hash(fn *image.Finder) (*types.Hash, error) {
 	exePath, err := os.Readlink("/proc/self/exe")
 	if err != nil {
 		if firstErr != nil {
-			return nil, fmt.Errorf("error finding stage1 images %q and %q in rkt binary directory: %v and %v", defaultStage1Image, imgBase, firstErr, err)
+			innerErr := errwrap.Wrap(firstErr, err)
```

Comment:
That is not exactly right, but probably errwrap has rather an error stack instead of an error tree.